react/features/chat/middleware.js

```diff
@@ -63,14 +68,17 @@ MiddlewareRegistry.register(store => next => action => {
             APP.API.notifyChatUpdated(unreadCount, isOpen);
         }
         break;
+
     case TOGGLE_CHAT:
-        unreadCount = 0;
-        isOpen = !getState()['features/chat'].isOpen;
+        isOpen = getState()['features/chat'].isOpen;
 
-        if (typeof APP !== 'undefined') {
-            APP.API.notifyChatUpdated(unreadCount, isOpen);
+        if (isOpen) {
```

Comment:
After your changes, is TOGGLE_CHAT used anywhere?

Reply:
It is still used by the web that actually has some "toggle functionality", pressing the chat button multiple times has this toggle effect.

Reply:
👍